Sources/SwiftDocCUtilities/Action/Actions/PreviewAction.swift

```diff
@@ -80,16 +73,7 @@ public final class PreviewAction: Action, RecreatingContext {
     
     /// Creates a new preview action from the given parameters.
     ///
-    /// The `tlsCertificateKey`, `tlsCertificateChain`, `serverUsername`,  and `serverPassword`
-    /// parameters are optional, but if you provide one, all four are expected. They are used by the preview server
-    /// to serve content on the local network over SSL.
-    ///
     /// - Parameters:
-    ///   - tlsCertificateKey: The path to the TLS certificate key used by the preview server for SSL configuration.
-    ///   - tlsCertificateChain: The path to the TLS certificate chain used by the preview server for SSL configuration.
-    ///   - serverUsername: The username used by the preview server for HTTP authentication.
-    ///   - serverPassword: The password used by the preview server for HTTP authentication.
-    ///   - port: The port number used by the preview server.
```

Comment:
minor: the `port` argument should still be documented

Reply:
Oh good catch- thanks!Resolved with https://github.com/apple/swift-docc/compare/c16e9e3a44c5258cfabec4393aab878deaa2af6e..fae61d1705b8562029372f5c6342c0b0eaddcd21.